components/fibery/sources/common/webhooks.mjs

```diff
@@ -0,0 +1,41 @@
+import fibery from "../../fibery.app.mjs";
+
+export default {
+  props: {
+    fibery,
+    db: "$.service.db",
+    http: "$.interface.http",
+    type: {
+      propDefinition: [
+        fibery,
+        "type",
+      ],
+    },
+  },
+  hooks: {
+    async deploy() {},
```

Comment:
is there any reason to leave an empty deploy function?

Reply:
No, I forgot to implement historical events